app/services/error-handler.js

```diff
@@ -1,29 +1,32 @@
-import classic from 'ember-classic-decorator';
-import Service from '@ember/service';
+import Service, { inject as service } from '@ember/service';
 
-@classic
 export default class ErrorHandlerService extends Service {
 
-  genericError(error) {
-    const errorStatus =  parseInt(error.errors[0].status);
-    if (errorStatus === 500) {
-      return 'There is an error error in server.';
-    } else if (errorStatus === 403) {
-      return 'You are not authorized to access the page.';
-    } else if (errorStatus === 400) {
-      return 'The server could not understand the request due to invalid syntax.';
-    } else if (errorStatus === 401) {
-      return 'The client must authenticate itself to get the requested response.';
-    } else if (errorStatus === 405) {
-      return 'The request method is known by the server but has been disabled and cannot be used.';
-    } else if (errorStatus === 409) {
-      return 'This response is sent when a request conflicts with the current state of the server.';
-    } else if (errorStatus === 422) {
-      return 'Unprocessable Entity';
-    } else if (errorStatus === 429) {
-      return 'Too many reuqests.';
-    } else {
-      return 'Opps Something went wrong.';
-    }
+@service notify;
+
+@service l10n;
+
+genericError(error) {
+  const errorStatus =  parseInt(error.errors[0].status);
+  switch (errorStatus) {
+    case 403:
+      this.notify.error(this.l10n.t('You are not authorized to access the page.'));
+      break;
+    case 422:
+      this.notify.error(this.l10n.t('Some required field is missing or is in wrong format.'));
+      break;
+    case 429:
+      this.notify.error(this.l10n.t('There are too many requests. Please try after sometime.'));
+      break;
+    case 401:
+      this.notify.error(this.l10n.t('User must be authenticated.'));
+      break;
+    case 409:
+      this.notify.error(this.l10n.t('Oops there is a conflict error'));
```

Comment:
User does not know what a conflict error is. Write `This item already exists on server`